examples/SSR/LICENSE

```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```

Comment:
This reminds me I should put a license into the repo so people that care about this stuff can use it :sweat_smile:, would you be ok with MIT? While I like the Unlicense it seems to give people headaches in certain jurisdictions and the JS library we are using here is also MIT licensed (totally fine for examples though imo).

Reply:
I just noticed I already did in the `Cargo.toml`, just didn't include a license file, fixing that. 

https://github.com/elsirion/leptos-qr-scanner/blob/48085be16cd8cea5130d7798a2d7793d96d7a8d2/Cargo.toml#L6

Reply:
Oh that's just what came with the starter template. I made no changes. Happy with MIT